src/main/resources/attackvectors/UnrestrictedFileUploadPayload.properties

```diff
@@ -0,0 +1,38 @@
+UNRESTRICTED_FILE_UPLOAD_PAYLOAD_LEVEL_2=By sending a file in proper format hacker can execute code in the user's browser, usually with elevated privileges (due to the fact that many users have an administrator account in Windows and disabled hardened browser policies as default). Typically, a persistent XSS attack is conducted to get sensitive cookie theft and sensitive data theft.<br/>\
+<b>How to exploit this level?</b><br/>\
+  Create a file with extension .html, and paste there some code, e.g.<br/>\
+  <code>&lt;html&gt;&lt;script&gt;alert(1);&lt;/script&gt;&lt;/html&gt;</code><br/>\
+  If we go to the URL where the file is stored, the code should be executed.
+UNRESTRICTED_FILE_UPLOAD_PAYLOAD_LEVEL_3=By sending a file in proper format hacker can execute code in the user's browser, usually with elevated privileges (due to the fact that many users have an administrator account in Windows and disabled hardened browser policies as default). Typically, a persistent XSS attack is conducted to get sensitive cookie theft and sensitive data theft.<br/>\
+<b>How to exploit this level?</b><br/>\
+  Filename validation was added, so now to exploit this level you have to create a file with extension .htm, and paste there some code, e.g.<br/>\
+  <code>&lt;html&gt;&lt;script&gt;alert(1);&lt;/script&gt;&lt;/html&gt;</code><br/>\
+  If we go to the URL where the file is stored, the code should be executed.
+UNRESTRICTED_FILE_UPLOAD_PAYLOAD_LEVEL_4=By sending a file in proper format hacker can execute code in the user's browser, usually with elevated privileges (due to the fact that many users have an administrator account in Windows and disabled hardened browser policies as default). Typically, a persistent XSS attack is conducted to get sensitive cookie theft and sensitive data theft.<br/>\
+<b>How to exploit this level?</b><br/>\
+  Filename validation was improved, so now to exploit this level you have to a create a file with extension .HTML, and paste there some code, e.g.<br/>\
+  <code>&lt;html&gt;&lt;script&gt;alert(1);&lt;/script&gt;&lt;/html&gt;</code><br/>\
+  If we go to the URL where the file is stored, the code should be executed.
+UNRESTRICTED_FILE_UPLOAD_PAYLOAD_LEVEL_5=By sending a file in proper format hacker can execute code in the user's browser, usually with elevated privileges (due to the fact that many users have an administrator account in Windows and disabled hardened browser policies as default). Typically, a persistent XSS attack is conducted to get sensitive cookie theft and sensitive data theft.<br/>\
+<b>How to exploit this level?</b><br/>\
+  Filename validation was improved (add case sensitive security), so now to exploit this level you have to create a file with extension .php, and paste there some code, e.g.<br/>\
```

Comment:
we can use xhtml or svg extension too and i think that will work in all the browsers

Reply:
I have tried on google chrome and script was not executed

Reply:
for xhtml and svg we need a different payload.